pyvista/plotting/qt_plotting.py

```diff
@@ -860,12 +866,18 @@ class MainWindow(QMainWindow):
     """Convenience MainWindow that manages the application."""
 
     signal_close = pyqtSignal()
+    signal_close_test = pyqtSignal()
 
     def __init__(self, parent=None):
         """Initialize the main window."""
         super(MainWindow, self).__init__(parent)
         self.signal_close.connect(self.close)
```

Comment:
This line needs to go - the event is emitted when the window is closed, so it will be an infinite loop. I assume this was mistakenly added instead of the emit originally.